recipes-core/packagegroups/packagegroup-middleware-layer.bb

```diff
@@ -172,7 +172,7 @@ RDEPENDS:${PN} = " \
     virtual/default-font \
     ${@bb.utils.contains('DISTRO_FEATURES', 'rdkwindowmanager', " rdkwindowmanager ", "", d)} \
     os-release \
-    ${@bb.utils.contains('DISTRO_FEATURES', 'ENABLE_MIRACAST', ' wlan-p2p', '', d)} \
+    wlan-p2p \
```

Comment:
Do we need to enable wlan-p2p for all platforms including STBs? Can we enable it for only TV platforms?